### DISCUSSION TITLE
removing google fonts

### DISCUSSION THREAD

**Discussion by hacknorris-code** [daattali/beautiful-jekyll#948] [Q&A] [answered]
so yes again me (and again with no reply ic)

so yes - Germany some time ago banned google fonts & analytics. for the now it possible to just not use analytics. how is this possible to remove fonts? what rule in prebuilt css is to remove google fonts fully? (saw it from privacy badger addon...) maybe im not in Germany but i dont want to give everyone 100€ after visit...

**Comment by daattali** [comment, marked as answer]
It seems they are imported here https://github.com/daattali/beautiful-jekyll/blob/a531737107167728c5fc8e25a4aadbd65d47533c/_layouts/base.html#L9-L10

In general, these typ of questions are best answered by doing a code search. That's exactly what I had to do - I just searched the codebase for "google". My personal advice is not to worry about this though.

**Reply by hacknorris-code** [reply]
in many countries in europe banned (or requests many money for developers)

**Reply by hacknorris-code** [reply]
but thanks anyways
